priv/repo/migrations/20220819164826_add_account_references_and_delete_behaviors_to_tables.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Ret.Repo.Migrations.AddAccountReferencesAndDeleteBehaviorToTables do
+  use Ecto.Migration
+
+  def change do
+    alter table(:oauth_providers) do
+      modify(:account_id, references(:accounts, column: :account_id, on_delete: :delete_all))
+    end
+
+    execute("alter table api_credentials drop constraint api_credentials_account_id_fkey")
+
+    alter table(:api_credentials) do
+      modify(:account_id, references(:accounts, column: :account_id, on_delete: :delete_all))
+    end
+  end
```

Comment:
The `change/0` signature is reserved for reversible migrations. Since the reverse cannot be inferred, you’ll want an `up/0` and `down/0` definition.